src/genie/ui/dialogs/color_map_editor.py

```diff
@@ -0,0 +1,196 @@
+import glob
```

Comment:
Remove unfinished code. Tag the branch, make an issue for possible future usage.

Reply:
This is where the color map is selected/imported. Should it be renamed?